src/sortRelatorTerms.js

```diff
@@ -0,0 +1,136 @@
+// Validator/fixer for sorting $e relator term subfields
+//
+// Author(s): Nicholas Volk
+
+import clone from 'clone';
+//import createDebugLogger from 'debug';
+import {fieldToString} from './utils';
+import {fieldFixPunctuation} from './punctuation2';
+//const debug = createDebugLogger('@natlibfi/marc-record-validators-melinda:sortRelatorTerms');
+//const debugData = debug.extend('data');
+
+const WORST_WORK = 99;
+
+const relatorTermValues = { // The higher, the better
+  // More abstract, the earlier it appears.
+  // Note that terms with same abstraction level might also have order preferences
+  // work/teos > expression/ekspressio > manifestation/manifestaatio > item/kappale
+  'säveltäjä': 100,
+  'kirjoittaja': 99, // Viola wants composer/säveltäjä on top (highly unlikely to ever appear together, but...)
+  'sanoittaja': 90,
+  // ekspressio
+  'sovittaja': 80,
+  'toimittaja': 80,
+  'editointi': 70,
+  'kääntäjä': 70,
+  // item/kappale
+  'esittäjä': 60,
```

Comment:
Possible additions and approximate values:

'taiteilija': 98,
'kuvittaja': 75,
'lukija': 60,